contributing.qmd

```diff
@@ -10,6 +10,15 @@ We encourage contributions to this guide. The guide's goal is to provide documen
 
 If you wish to preview the site locally, install [quarto](https://quarto.org/). You will also need to be familiar with [quarto markdown](https://quarto.org/docs/authoring/markdown-basics.html).
 
+## Contribution Workflow
```

Comment:
Having this section near the top is probably the right spot so it doesn't get lost with the other sections.